dashboard/app/models/course_version.rb

```diff
@@ -42,15 +40,15 @@ class CourseVersion < ApplicationRecord
   UNVERSIONED = 'unversioned'.freeze
 
   def units
-    content_root_type == 'UnitGroup' ? content_root.default_units : [content_root]
+    content_root.default_units
   end
 
   # "Interface" for content_root:
   #
   # is_course? - used during seeding to determine whether this object represents the content root for a CourseVersion.
   #   For example, this should return True for the CourseA-2019 Unit and the CSP-2019 UnitGroup. This should return
   #   False for the CSP1-2019 Unit.
```

Comment:
remove outdated comment? some of what's here is inaccurate, and the whole idea of commenting this interface seems somewhat unnecessary now that this always points to a UnitGroup.